### DISCUSSION TITLE
Frequent reboots - MQTT counts in the 1000's and my solution

### DISCUSSION THREAD

**Discussion by bazmk2** [arendst/Tasmota#14503] [Q&A] [answered]
The short version:

I had a NTP server in my DHCP scope parameters served to Tasmota that on update from 9.x to 10.x broke my devices.  MQTT counts in the 100's, uptimes in minutes.  Removed NTP from DHCP scope and devices all start behaving again.

The long version:

I had a stable Home assistant running as a VM with a remote OrangePi as the zwave and zigbee bridge using HA and zigbee2mqtt.  All was fine until I decided to go on an update spree.  I updated Home Assistant, the mosquitto addon and also the majority of my tasmota devices.  Used the tasmota admin add on in HA and one click and they all OTA update.  Feeling satified that I had up to date systems I sat back.

Of course it went wrong.  I began to experience very unstable systems, HA would lose devices, became unresponsive.  Lost my OPi bridge as zigbee became unusable.  frequent MQTT reconnects and drops in the mosquitto broker.  Spent the day installing, uninstalling, setting up from scratch, implementing the whole HA on the OPi and this is where I left it as relatively stable but was still losing devices and MQTT counts where in the 1000's.  Put it down to 'some error' that i hoped would sort itself out.

Yesterday I solved it.  I opened up Tasmota-Admin addon and noticed that everything on 10.x has an uptime of minutes.  One device that had stayed on 9.x (i think 9.1) had an uptime of days.  WHAT???  did a bit more googling and found some issues with DHCP scopes and the code changes underlying the 10.x from 9.1.  Looked at my DHCP scope for this and lo and behold I had set an NTP server in my DHCP options some time (no pun) ago.  Removed this and all of the tasmotas are back to normal.

I have learnt that I can set up and use HiveMQ too!  One of my troubleshooting actions was to use a different MQTT broker.

Interstingly there was no issues with the NTP server - it is synched, serves the correct time to other devices etc so I am not sure what the issue was but in case anyone has a similar issue I hope I can help.

**Comment by sfromis** [comment, marked as answer]
Since some time Tasmota no longer attempts to use NTP server config from DHCP, you *have to* use the command `NtpServer` to configure for the up to 3 possible servers (unless you want to use the default public ones). And without NTP server access, Tasmota runs very badly.

**Reply by bazmk2** [reply]
If the device doesn't use DHCP provided NTP why does it cause it to reboot?  And then return to normal when it is removed.  Seems to be a change from 9.1?  

NTP access is open on my network.

**Reply by sfromis** [reply]
I don't know what happens, good crystal balls are out of stock. 9.1 was the last version to try to use DHCP announced NTP servers, so definitely a change since then.

**Reply by bazmk2** [reply]
LOL if you find a supply of any future prediction devices aka crystal ball hit me up - but only if I can flash them……..

Anyway I struggled to find an answer to my issue so hope this helps someone else :)
